data/state/accountsDB.go

```diff
@@ -962,8 +962,9 @@ func (adb *AccountsDB) SnapshotState(rootHash []byte) {
 	go func() {
 		stopWatch := core.NewStopWatch()
 		stopWatch.Start("snapshotState")
-		trieStorageManager.TakeSnapshot(rootHash, createNewDb)
-		adb.snapshotUserAccountDataTrie(rootHash, true)
+		leavesChannel := make(chan core.KeyValueHolder, 100)
```

Comment:
why 100? can you extract a constant at least?

Reply:
Copied it from GetAllLeavesOnChannel :D  Extracted it in a const.